packages/yew/src/html/conversion.rs

```diff
@@ -83,10 +83,17 @@ macro_rules! impl_into_prop {
 
 // implemented with literals in mind
 impl_into_prop!(|value: &'static str| -> String { value.to_owned() });
+impl_into_prop!(|value: &'static str| -> Cow<'static, str> { Cow::Borrowed(value) });
```

Comment:
I would rather not have this. `AttrValue` should be used instead of `Cow<'static, str>`

A `From` implementation for `AttrValue` would be fine to have, if there's really a need for using `Cow<'static, str>`

Reply:
Hi! The use case is described in the description as second example.
I did a simple search box that takes a "placeholder" parameter which was a String, then I read #1843 so I tried to use Cow<'static, str> instead of String. Without implementing IntoPropValue<Cow<'static, str>> for 'str, I got an error suggesting to implement IntoPropValue<Cow<'static, str>> for 'str when doing:
`<Search placeholder="some text" />` 
I would also do it this way:
`<Search placeholder={ Cow::from("some text") }` but this seems a bit disapointing :)
So I will remove if this is something not acceptable, it's not a deal breaker to me! 
Let me know what you think.